src/Components/WebAssembly/Sdk/testassets/blazorwasm-fxref/Program.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace standalone
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+        }
+    }
+}
```

Comment:
I appreciate the test asset being so minimal! Much better than duplicating the project template again.